### PR TITLE
MCLOUD-6430: Xdebug not working on Magento Cloud Docker 1.1

### PR DESCRIPTION
**Pull request by oshmyheliuk**
## Purpose of this pull request

This pull request (PR) will update information on how to use xdebug with docker on Linux systems

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/docker/docker-development-debug.html

whatsnew
Added [instructions](https://devdocs.magento.com/cloud/docker/docker-development-debug.html) for using Xdebug with Docker on Linux systems.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-development-debug.md [hunk 1]
```diff
@@ -23,8 +23,14 @@ If you use Microsoft Windows, take the following steps before continuing:
 1. To enable Xdebug for your Docker environment, generate the Docker Compose configuration file in developer mode with the `--with-xdebug` option and any other required options, for example.
 
    ```bash
-   vendor/bin/ece-docker build:compose --mode --sync-engine="mutagen" developer --with-xdebug
+   vendor/bin/ece-docker build:compose --mode="developer" --sync-engine="mutagen" --with-xdebug
    ```
+   
+   For Linux system also need to be added option `--set-docker-host` for automatically adding `host.docker.internal` into `/etc/hosts` of `fpm_xdebug` container.
```
Comment (BarnyShergold) [inline src/cloud/docker/docker-development-debug.md:29]: Better English syntax
```suggestion
   For Linux systems, the option `--set-docker-host` also needs to be added for automatically adding `host.docker.internal` into the `/etc/hosts` for the `fpm_xdebug` container.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/cloud/docker/docker-development-debug.md (hunk 1 above)
Comment (oshmyheliuk) [reply src/cloud/docker/docker-development-debug.md]: Thank you!

**Review by BarnyShergold — APPROVED** [review]
No problems!

File: src/cloud/docker/docker-development-debug.md [hunk 2]
```diff
@@ -23,8 +23,14 @@ If you use Microsoft Windows, take the following steps before continuing:
 1. To enable Xdebug for your Docker environment, generate the Docker Compose configuration file in developer mode with the `--with-xdebug` option and any other required options, for example.
 
    ```bash
-   vendor/bin/ece-docker build:compose --mode --sync-engine="mutagen" developer --with-xdebug
+   vendor/bin/ece-docker build:compose --mode="developer" --sync-engine="mutagen" --with-xdebug
    ```
+   
+   For Linux systems, the option `--set-docker-host` also needs to be added for automatically adding `host.docker.internal` into the `/etc/hosts` for the `fpm_xdebug` container.
```
Comment (jeff-matthews) [inline src/cloud/docker/docker-development-debug.md:29]:
```suggestion
   For Linux systems, you must use the `--set-docker-host` option to add the `host.docker.internal` entry to the `/etc/hosts` file for the `fpm_xdebug` container.
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

File: src/cloud/docker/docker-development-debug.md (hunk 2 above)
Comment (BarnyShergold) [reply src/cloud/docker/docker-development-debug.md:29]: So it's a compulsory option?

Comment (jeff-matthews) [reply src/cloud/docker/docker-development-debug.md:29]: The original text implies that it is.

**Review by jeff-matthews — APPROVED** [review]

**Review by meker12 — APPROVED** [review]

**Pull request merged**